doc/source/terrain.md

```diff
@@ -20,20 +113,63 @@ The slope of a DEM describes the tilt, or gradient, of each pixel in relation to
 It is most often described in degrees, where a flat surface is 0° and a vertical cliff is 90°.
 No tilt direction is stored in the slope map; a 45° tilt westward is identical to a 45° tilt eastward.
 
-The slope can be computed either by the method of [Horn (1981)](http://dx.doi.org/10.1109/PROC.1981.11918) (default)
-based on a refined gradient formulation on a 3x3 pixel window, or by the method of [Zevenbergen and Thorne (1987)](http://dx.doi.org/10.1002/esp.3290120107) based on a plane fit on a 3x3 pixel window.
+The slope $\alpha$ can be computed either by the method of [Horn (1981)](http://dx.doi.org/10.1109/PROC.1981.11918) (default)
+based on a refined gradient formulation on a 3x3 pixel window, or by the method of
+[Zevenbergen and Thorne (1987)](http://dx.doi.org/10.1002/esp.3290120107) based on a plane fit on a 3x3 pixel window.
+
+For both methods, $\alpha = arctan(\sqrt{p^{2} + q^{2}})$ where $p$ and $q$ are the gradient components west-to-east and south-to-north, respectively, with for [Horn (1981)](http://dx.doi.org/10.1109/PROC.1981.11918):
+
+$$
+p_{\textrm{Horn}}=\frac{(h_{++} + 2h_{+0} + h_{+-}) - (h_{-+} + 2h_{-0} + h_{--})}{8 \Delta x},\\
+q_{\textrm{Horn}}=\frac{(h_{++} + 2h_{0+} + h_{-+}) - (h_{+-} + 2h_{0-} + h_{--})}{8 \Delta y},
+$$
+
+and for [Zevenbergen and Thorne (1987)](http://dx.doi.org/10.1002/esp.3290120107):
+
+$$
+p_{\textrm{ZevTho}} = \frac{h_{+0} - h_{-0}}{2 \Delta x},\\
+q_{\textrm{ZevTho}} = \frac{h_{0+} - h_{0-}}{2 \Delta y},
+$$
+
+where $h_{ij}$ is the elevation at pixel $ij$, where indexes $i$ and $j$ correspond to east-west and north-south directions respectively,
+and take values of either the center ($0$), west or south ($-$), or east or north ($+$):
+
+```{list-table}
+   :widths: 1 1 1 1
+   :header-rows: 1
+   :stub-columns: 1
+
+   * -
+     - West
+     - Center
+     - East
+   * - North
+     - $h_{-+}$
+     - $h_{0+}$
+     - $h_{++}$
+   * - Center
+     - $h_{-0}$
+     - $h_{00}$
+     - $h_{+0}$
+   * - South
+     - $h_{--}$
+     - $h_{0-}$
+     - $h_{+-}$
+```
 
-The differences between methods are illustrated in the {ref}`sphx_glr_basic_examples_plot_terrain_attributes.py`
-example.
 
-```{image} basic_examples/images/sphx_glr_plot_terrain_attributes_001.png
-:width: 600
-```
+Finally, $\Delta x$
+and $\Delta y$ correspond to the pixel resolution west-east and south-north, respectively.
```

Comment:
Add a note that we do not support non square pixels in xDEM at the moment? It will raise an error.

Reply:
Added